src/unix/bsd-ifaddrs.c

```diff
@@ -119,14 +119,19 @@ int uv_interface_addresses(uv_interface_address_t** addresses, int* count) {
       continue;
 
     address = *addresses;
-    memset(address->phys_addr, 0, sizeof(address->phys_addr));
```

Comment:
Aside 1: this memset() was an out-of-bounds write when `*count == 0`. linux-core.c returns early when `*count == 0` but this file does not.

Aside 2: `*count == 0` also introduces platform/libc-specific behavior due to the zero-sized allocation; that either returns NULL or a unique pointer.

Reply:
> Aside 1: this memset() was an out-of-bounds write when *count == 0. linux-core.c returns early when *count == 0 but this file does not.

Yea, I noticed that yesterday. I was going to send a follow up PR for that. I could just add a commit to this PR if you'd prefer.

Reply:
Either is fine, no preference.